.travis.yml

```diff
@@ -11,9 +11,11 @@ before_install:
     - 'if [[ $GROUP != js* ]]; then COVERAGE=""; fi'
 install:
     - pip install -f travis-wheels/wheelhouse -r requirements.txt -e file://$PWD#egg=ipython[test] coveralls
+    - pip install codecov
```

Comment:
Yeah, I've seen some intermittent failures only on 3.5 recently on other repos in the past few days, as well. I didn't mean to imply that *you* made no sense, only that Travis' behavior makes no sense, as is often the case.